doc/contributing/releases.md

```diff
@@ -698,10 +698,16 @@ placeholders (that happens when a change previously landed on another release
 branch), keep both version numbers. Convert the YAML field to an array if it is
 not already one.
 
+Note: [It's possible that the `cherry-pick` step will end up adding and/or
```

Comment:
Totally optional, but I would avoid "Note:" unless it's really necessary (which is almost never is).

```suggestion
[It's possible that the `cherry-pick` step will end up adding and/or
```